libr/include/r_bp.h

```diff
@@ -48,6 +48,7 @@ typedef struct r_bp_item_t {
 	int size; /* size of breakpoint area */
 	int recoil; /* recoil */
 	bool swstep; 	/* is this breakpoint from a swstep? */
+	bool valid; /* Whether the breakpoint in a valid map */
```

Comment:
this is kind of cache, because the real value can change at any time, wouldnt be better to have an api to determine if its in a valid map or not?

Reply:
Removed it and replaced with calls to r_bp_is_valid whenever it needs to be verified.